packages/nuxt/src/core/nitro.ts

```diff
@@ -415,3 +402,24 @@ export async function initNitro (nuxt: Nuxt & { _nitro?: Nitro }) {
 function relativeWithDot (from: string, to: string) {
   return relative(from, to).replace(/^([^.])/, './$1') || '.'
 }
+
+function spaLoadingTemplate (options: Nuxt['options']) {
```

Comment:
Totally minor point, but let's take `nuxt` as the argument, for consistency with other code.